integ/workflow/wf_state_options_data_attributes_loading/routers.go

```diff
@@ -0,0 +1,292 @@
+package wf_state_options_data_attributes_loading
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/integ/workflow/common"
+	"github.com/indeedeng/iwf/service"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"net/http"
+)
+
+/**
+ * This test has four states
```

Comment:
Nice! I planned to add documentation to all the exsting integ test workflow, and wondering what is the format. I think it's a great idea that just put it at the beginning.

Nit: `This test has four states` -> `This test workflow has four states, using REST controller to implement the workflow directly.`

Reply:
Will change! Thanks!

Do you think that is best for the comment to be at the start of the routers file (where it currently is) or at the start of the test file (i.e. `wf_state_options_data_attributes_loading_test.go`)?

Reply:
Done ✅